### PR TITLE
fix(security): Clean up Consul token generation code from security-bootstrapper

### PR DESCRIPTION
**Pull request by jim-wang-intel**
With the latest go-mod-bootstrap code to generate Consul tokens for each service,
we don't need the old way of generating consul tokens any more.

Clean all codes related to that and also remove the configuration items related to that.

Closes: #3348

Signed-off-by: Jim Wang <yutsung.jim.wang@intel.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/master/.github/Contributing.md.

## What is the current behavior?

Current code has unnecessary token-generation code from `security-bootstrapper`

## What is the new behavior?
Consul token generation code removed.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x ] No

## New Imports

- [ ] Yes
- [x ] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: internal/security/bootstrapper/command/setupacl/command.go [hunk 1]
```diff
@@ -184,24 +177,6 @@ func (c *cmd) GetCommandName() string {
...
 
-func (c *cmd) getSecretStoreClient() (secrets.SecretStoreClient, error) {
-	clientConfig := types.SecretConfig{
-		Type:     c.configuration.SecretStore.Type,
-		Protocol: c.configuration.SecretStore.Protocol,
-		Host:     c.configuration.SecretStore.Host,
-		Port:     c.configuration.SecretStore.Port,
-	}
-
-	secretstoreClient, err := secrets.NewSecretStoreClient(clientConfig, c.loggingClient, c.client)
```
Comment (lenny-intel) [inline internal/security/bootstrapper/command/setupacl/command.go:195]: With this removed, the GenerateConsulToken can now be removed from SecretStoreClient interface in go-mod-secrets, correct. And its `token` parameter can be removed and implementation updated , i.e. now only uses the service vault token.

**Review by lenny-intel — APPROVED** [review]
LGTM

File: internal/security/bootstrapper/command/setupacl/command.go (hunk 1 above)
Comment (jim-wang-intel) [reply internal/security/bootstrapper/command/setupacl/command.go:195]: Right. Good point, i'll created an issue for that tech-debt.

Comment (jim-wang-intel) [reply internal/security/bootstrapper/command/setupacl/command.go:195]: issue https://github.com/edgexfoundry/go-mod-secrets/issues/104 created

**Review by bnevis-i — APPROVED** [review]

**Pull request merged**